### DISCUSSION TITLE
Two-column teaser

### DISCUSSION THREAD

**Discussion by taiya** [cvpr-org/author-kit#27] [Ideas]
![Screenshot by Dropbox Capture](https://github.com/cvpr-org/author-kit/assets/4480573/95b85026-d0c3-4b1c-bde2-2cf0dda5923e)

Replace this
```
\maketitle
```

with this
```
\twocolumn[{%
\renewcommand\twocolumn[1][]{#1}%
\maketitle
\includegraphics[width=.32\linewidth]{example-image-a.png}
\includegraphics[width=.32\linewidth]{example-image-b.png}
\includegraphics[width=.32\linewidth]{example-image-c.png}
% \vspace{-2em}
\captionof{figure}{Caption of the teaser \vspace{1em}}
\label{fig:teaser}
}]
```

**Comment by igkiou** [comment]
Given two-column teasers are becoming increasingly popular in CVPR/ICCV, it's probably worth wrapping this inside a dedicated command, e.g., `teaser`, similar to the one in the SIGGRAPH template (`acmtog` ).

**Reply by taiya** [reply]
Agreed, feel free to submit a PR with a proposed change.

**Reply by taiya** [reply]
Also, while at siggraph this is the standard (two col teaser), at CVPR it is not (only some people use it).
So I am not sure how I could get this done in a way that makes everybody happy.

But again, feel free to submit a PR for it!

**Comment by XuYunqiu** [comment]
Hi, is there any way to add captions of the subfigures in the two-column teaser

**Reply by taiya** [reply]
You can probably get it done with latex's minipages within the twocol (never tried though).
